src/databricks/labs/remorph/transpiler/sqlglot/lca_utils.py

```diff
@@ -15,20 +15,26 @@
 
 
 def check_for_unsupported_lca(
-    dialect: DialectType,
-    sql: str,
-    filename: str,
+    from_dialect: str,
```

Comment:
Any reason we can't propagate dialect type directly here?
```suggestion
    from_dialect: DialectType,
```

This will avoid cyclic imports

Reply:
Agreed, done